bee-message/src/output/mod.rs

```diff
@@ -198,3 +200,12 @@ impl Output {
         }
     }
 }
+
+impl ByteCost for Output {
+    fn weighted_bytes(&self, config: &ByteCostConfig) -> u64 {
+        // The updated verison of TIP19 (https://github.com/iotaledger/tips/pull/39) has been largely simplified.
```

Comment:
I wouldn't mention a temporary past version tbh.

Reply:
Reopening, just making sure a change wasn't lost

Reply:
Sorry, it makes sense to save open files before committing.